app/src/main/java/io/github/droidkaigi/confsched/model/Session.java

```diff
@@ -85,6 +85,27 @@
     public Session() {
     }
 
+    /***
+     * The copy constructor
+     * @param session copy from
+     */
+    public Session(Session session) {
```

Comment:
I know what you mean...
As you said, session object is not immutable. But I'm also not sure this is right way 😢